arangod/Cluster/ClusterInfo.cpp

```diff
@@ -229,6 +241,48 @@ void ClusterInfo::cleanup() {
   theInstance->_currentCollections.clear();
 }
 
+void ClusterInfo::triggerBackgroundGetIds() {
+  // Trigger a new load of batches
+  _uniqid._nextBatchStart = 1ULL;
+  _uniqid._nextUpperValue = 0ULL;
+
+
+  try {
+    if (_uniqid._backgroundJobIsRunning) {
+      return ;
+    }
+    _uniqid._backgroundJobIsRunning = true;
+    std::thread([this]{
+      auto guardRunning = scopeGuard([this]{
+        MUTEX_LOCKER(mutexLocker, _idLock);
+        _uniqid._backgroundJobIsRunning = false;
+      });
+
+      uint64_t result;
+      try {
+        result = _agency.uniqid(MinIdsPerBatch, 0.0);
+      } catch (std::exception const& e) {
```

Comment:
this will cause a warning about an unused variable `e` in MSVC.

Reply:
This has to be fixed for windows, agreed.